common/ipmi/ipmb.c

```diff
@@ -704,7 +701,7 @@ void IPMB_SeqTimeout_handler(void *arug0, void *arug1, void *arug2)
         continue;
       }
 
-      osMutexAcquire(mutex_id[index], osWaitForever);
+      k_mutex_lock(&mutex_id[index], K_FOREVER);
```

Comment:
Timeout vs. Forever?

Reply:
Agree with making the same change as is mentioned in insert_node case comment.